docs/data/export.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Export Data"
-linkTitle: "Export Data"
-description: "Export data from the Viam app to your local computer using the Viam CLI."
+title: "Upload and Export Data"
+linkTitle: "Upload and Export Data"
+description: "Download data from and upload data to the Viam app from your local computer using the data client API, Viam CLI, or Viam mobile app."
```

Comment:
I think we should create upload as its own page - up to you and @JessamyT whether you want to merge this PR and create the new page later or create the page now.

Reply:
I would be good with creating the new page later 😄

Reply:
I think that does make sense because by itself this may look odd

Reply:
can you comment this pr or the page link on the jira ticket for the upload page?

Reply:
@npentrel Sorry, I can't find that ticket! Which one are you referring to?